### DISCUSSION TITLE
Integrating XDC Pay to Rainbow Kit

### DISCUSSION THREAD

**Discussion by deeluckystar** [rainbow-me/rainbowkit#1432] [Q&A] [answered]
Just wondering if there is an easy way to integrate Xdc Pay into Rainbow Kit so users can also select XDC wallet if they wish

**Comment by DanielSinclair** [comment, marked as answer]
You can use the Custom Wallets API documented [here](https://www.rainbowkit.com/docs/custom-wallets) to add custom wallets to your dApp. Reference of internal connector implementations [here](https://github.com/rainbow-me/rainbowkit/tree/main/packages/rainbowkit/src/wallets/walletConnectors) for more examples.
